asciidoc/getting_started/azure-node/app-insights.adoc

```diff
@@ -0,0 +1,50 @@
+== @ensono-stacks/azure-node:app-insights
+
+Installs and configures App Insights in your NodeJS app.
+
+This generator will add and configure https://www.npmjs.com/package/applicationinsights[applicationinsights] npm package for you.
+
+=== Prerequisites
+
+- Requires a NodeJs server for application insights to hook into.
+
+TIP: Use the https://nx.dev/packages/next/generators/custom-server[@nx/next:custom-server] generator which will have been added to your workspace by xref:plugin-information.adoc[@ensono-stacks/workspace].
+
+/docs/getting_started/workspace/ensono-stacks-workspace[@ensono-stacks/workspace] to generate your NodeJS server!
```

Comment:
When using a relative path like this, i think you need to use `link:` before it or it doesn't render.
```
link:/docs/getting_started/workspace/ensono-stacks-workspace[@ensono-stacks/workspace]
```

Reply:
Sure, I forgot about this